scripts/sanitycheck

```diff
@@ -2792,9 +2792,14 @@ def parse_arguments():
         action="store",
         help="Save list of tests to be run to file.")
 
-    parser.add_argument(
+    group = parser.add_mutually_exclusive_group()
```

Comment:
```python
test_xor_build = parser.add_mutually_exclusive_group()
```
... or any more specific name than "group" so add_mutually_exclusive_group() can be used again elsewhere if needed.